client/components/plans/plan-actions/index.jsx

```diff
@@ -188,10 +209,12 @@ module.exports = React.createClass( {
 	},
 
 	freePlanExpiration: function() {
-		if ( ! this.planHasCost() ) {
-			return (
-				<span className="plan-actions__plan-expiration">{ this.translate( 'Never expires', { context: 'Expiration info for free plan in /plans/' } ) }</span>
-			);
+		if ( config.isEnabled( 'upgrades/free-trials' ) ) {
+			if ( ! this.planHasCost() ) {
```

Comment:
Super minor, but we should probably combine these conditions:
`config.isEnabled( 'upgrades/free-trials' ) && ! this.planHasCost()`